### PR TITLE
release 1.0.0 SDK-105

### PR DESCRIPTION
**Pull request by piyush-kukadiya**

### CODE REVIEW

File: clevertap-geofence-sdk/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    <application>
+
+        <receiver android:name=".CTGeofenceReceiver">
+        </receiver>
+
+        <receiver android:name=".CTLocationUpdateReceiver">
+        </receiver>
+
+        <receiver android:name=".CTGeofenceBootReceiver">
```
Comment (darshanclevertap) [inline clevertap-geofence-sdk/src/main/AndroidManifest.xml:17]: Add android:exported="false"

File: README.md [hunk 2]
```diff
@@ -1 +1,209 @@
...
+`CTGeofenceSettings` object can be created in the following way -
+
+```java
+CTGeofenceSettings ctGeofenceSettings = new CTGeofenceSettings.Builder()
+                .enableBackgroundLocationUpdates(bgLocation)//boolean to enable background location updates
+                .setLogLevel(logLevel)//Log Level
+                .setLocationAccuracy(accuracy)//byte value for Location Accuracy
+                .setLocationFetchMode(fetchMode)//byte value for Fetch Mode
+                .setGeofenceMonitoringCount(geofenceCount)//int value for number of Geofences CleverTap can monitor
+                .setInterval(interval)//long value for interval
```
Comment (kumaratulismu) [inline README.md:76]: Please add "long value for interval in millis" in comment for **setInterval**.

File: app/proguard-rules.pro [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+#   public *;
+#}
+
+# Uncomment this to preserve the line number information for
+# debugging stack traces.
+#-keepattributes SourceFile,LineNumberTable
+
+# If you keep the line number information, uncomment this to
+# hide the original source file name.
+#-renamesourcefileattribute SourceFile
```
Comment (kumaratulismu) [inline app/proguard-rules.pro:21]: Please add our proguard rules

-keep class com.google.android.gms.common.*
-keep class com.google.android.gms.location.*
-keep class androidx.concurrent.futures.*

**Review by kumaratulismu — CHANGES_REQUESTED** [review]
Please add the changes

File: README.md (hunk 2 above)
Comment (piyush-kukadiya) [reply README.md]: done

File: app/proguard-rules.pro (hunk 3 above)
Comment (piyush-kukadiya) [reply app/proguard-rules.pro:21]: done

**Review by darshanclevertap — APPROVED** [review]
Let's release this 👍

**Review by kumaratulismu — APPROVED** [review]
Looks fine

**Pull request merged**